src/www/app/cordova_main.ts

```diff
@@ -164,6 +165,10 @@ class CordovaPlatform implements OutlinePlatform {
     return new AbstractUpdater();
   }
 
+  getVpnServiceInstaller(): VpnInstaller {
+    return new NoOpVpnInstaller();
```

Comment:
This could be a place to request for VPN permission and install the VPN profile on mobile in the future. I'm not sure we do the "heads up" before the permission prompt currently.